terraform/amazon/templates/instance_pools/instance_pool_instance.tf.template

```diff
@@ -124,6 +123,15 @@ resource "aws_autoscaling_group" "{{.TFName}}" {
     propagate_at_launch = true
   }
 }
+
+resource "aws_autoscaling_policy" "{{.TFName}}-autoscaling-policy" {
```

Comment:
As far as I see is the scaling policy doing exactly nothing, let me know if that is not true? I would merge this after we remove the scaling policy...

Reply:
@simonswine Yep, spot on. It is indeed superfluous.  Removed as requested.